microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/install.md

```diff
@@ -0,0 +1,285 @@
+
+1. Go to [MicrosoftEdge/Demos repo](https://github.com/MicrosoftEdge/Demos/).
+
+1. If the green **Code** button isn't shown, click **Demos** in the path **Microsoft Edge** / **Demos** in the upper left to go to the main page of the repo.
+
+1. Click the green **Code** button, and then select the **Copy** button.  This article assumes you're using the **Source Control** features within Visual Studio Code.  Or, you can use one of the other methods that are provided, if you know that you want to use that approach instead:
+   *  **Open with GitHub Desktop**
+   *  **Open with Visual Studio**
+   *  **Download ZIP**
+
+   ![Cloning the MicrosoftEdge/Demos repo](./get-started-images/clone-repo.png)
+
+1. In Visual Studio Code, in the Activity Bar, click the **Source Control** (![Source Control icon](./get-started-images/source-control-icon.png)) button, and then click the **Clone Repository** button.
+
+1. In the **Provide repository URL** text box, paste the copied URL: **https://github.com/MicrosoftEdge/Demos.git** and then press `Enter`.  A folder selection dialog opens.
+
+   ![The Clone Repository button in Visual Studio Code](./get-started-images/clone-repository-button.png)
+
+1. Navigate to your desired path, such as `C:\Users\myusername\Documents\GitHub` or `Users/myusername/GitHub`, and then click the **Select Repository Location** button.
+
+1. The message **Cloning git repository** appears, then you're prompted to open the cloned repository.  Click the **Open** button:
+
+   ![Prompted to open the cloned repo](./get-started-images/prompt-open-cloned-repo.png)
+
+1. If prompted **Do you trust...**, click the **Yes** button.  Or, click the **No** button and continue with most parts of this walkthrough.
+
+   The **Explorer** tree lists many demos, including **demo-to-do**.
+
+1. In Visual Studio, select **File** > **Close Folder**.
+
+1. In Visual Studio Code, in the Activity Bar, click the **Microsoft Edge Tools** (![Microsoft Edge Tools icon](./get-started-images/microsoft-edge-tools-icon.png)) button.  The **Microsoft Edge Tools** pane opens.
+
+1. In the **Microsoft Edge Tools** > **Targets** pane, click the **Open Folder** button.  The **Open Folder** dialog box opens.  Navigate to the `demo-to-do` folder in the Demo repo that you cloned, select the folder or go into it, and then click the **Select Folder** button:
+
+   ![Selecting the demo-to-do folder](./get-started-images/select-demo-to-do-folder.png)
+
+   An example of a repo location where the **Demos** repo has been cloned is shown above.  The `demo-to-do` folder of the cloned **Demos** repo opens in the Explorer of Visual Studio Code, and no `launch.json` file exists yet:
+
+   ![Opened the demo-to-do folder initially](./get-started-images/opened-demo-to-do-folder-initially.png)
+
+
+<!-- ====================================================================== -->
+## Step 6: Set up a localhost server by using npx http-server
```

Comment:
As it makes no sense for users to install http-server and run and IIS server, should we roll step 6 and 7 into one and offer these as alternatives to each other?

Reply:
Seems like a clean easy way to do that is:

h2: Step 6: Set up a localhost server
h4: Set up a localhost server by using npx http-server
h4: Set up a localhost server by using IIS

might draft & push that